src/datumaro/plugins/data_formats/datumaro/base.py

```diff
@@ -115,7 +115,12 @@ def _load_categories(parsed) -> Dict:
         if parsed_points_cat:
             point_categories = PointsCategories()
             for item in parsed_points_cat["items"]:
-                point_categories.add(int(item["label_id"]), item["labels"], joints=item["joints"])
+                point_categories.add(
+                    int(item["label_id"]),
+                    item["labels"],
+                    joints=item["joints"],
+                    positions=item.get("positions"),
```

Comment:
`item.get("...")` looks a bit inconsistent vs `item["..."]`. Is there a specific reason the mix different styles of element access?

Reply:
This is to ensure backward compatibility. Datasets that were created with older versions of Datumaro may not have the "positions" key, so retrieving them directly using `item["positions"]` would result in an error. By soft-getting the positions value, we ensure that we can safely load these older datasets.